src/main.js

```diff
@@ -5,21 +5,25 @@ var UTF8 = "utf-8";
 
 var pkg = require("../package.json");
 var fs = require("fs");
+var vm = require("vm");
 var chalk = require("chalk");
+var hook = require("node-hook");
+var path = require("path");
 var uniq = require("lodash/array/uniq");
 var nomnom = require("nomnom")
     .script("squiggle")
     .option("input", {
         position: 0,
         metavar: "FILE",
-        help: "Compile this Squiggle file",
+        help: "Execute this Squiggle file",
         type: "string"
     })
     .option("output", {
-        position: 1,
         metavar: "FILE",
+        abbr: "o",
         help: "Write JavaScript to this file",
-        type: "string"
+        type: "string",
+        flag: true
```

Comment:
This shouldn't be a `flag` option. It should be be a regular parameter with a value `-o FILE`. I'm fine if this breaks existing calls to `squiggle main.sqg out.js`.